src/os/KeyValueStore.h

```diff
@@ -217,7 +217,7 @@ class KeyValueStore : public ObjectStore,
 
   string strip_object_key(uint64_t no) {
     char n[100];
-    snprintf(n, 100, "%lld", (long long)no);
+    snprintf(n, 100, "%08lld", (long long)no);
```

Comment:
This change is to keep the key ordering. order of keys are depends on the ascii value, so if we have keys from key_1 to key_99, order will look like key_1, key_11, .., key_2, key_20, ... etc.. we might not fit all the keys in a node. hence modified to have preserve the locality of reference.